test/Microsoft.Web.LibraryManager.Vsix.Test/UI/Models/LibraryIdViewModelTests.cs

```diff
@@ -18,7 +18,17 @@ public class LibraryIdViewModelTests
         private readonly ILibraryCatalog _testCatalog = new LibraryCatalog()
             .AddLibrary(new Library { Name = "test", Version = "1.0" })
             .AddLibrary(new Library { Name = "test", Version = "1.2" })
-            .AddLibrary(new Library { Name = "test", Version = "2.0" });
+            .AddLibrary(new Library { Name = "test", Version = "2.0" })
+            .AddLibrary(new Library { Name = "@types/node", Version = "1.0" })
```

Comment:
Names like "@types/node" are real package name, so it can be confusing as to whether something is going to hit a live endpoint or not to get data about this.  There's no benefit to using a real package name over using a fake name (unless you're doing an integration test).